src/block/icon-list/edit.js

```diff
@@ -255,9 +274,9 @@ const Edit = props => {
 					/>
 
 					<AdvancedSelectControl
-						label={ __( 'List Type', i18n ) }
+						label={ __( 'List Style Type', i18n ) }
```

Comment:
Revert the string so we retain the translations
```suggestion
						label={ __( 'List Type', i18n ) }
```